src/metadata-container-json-schema.json

```
@@ -6,7 +6,7 @@
     "properties": {
         "document_version": {
             "$comment": "Version of this model",
-            "const": "1.0.0"
+            "const": "1.0.1"
```

```suggestion
            "const": "1.0.0"
```
Normally I only bump this if there's a change to the structure in this file, not because of changes in underlying models.

---

That makes sense, I'll revert the change